addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  updateModel(model, infinityModel) {
```

Comment:
Are you saying that when overriding, people should use `_modelPath` to get the `infinityModel`?